src/test/java/com/smartcar/sdk/VehicleTest.java

```diff
@@ -146,9 +149,11 @@ public void testLock() throws Exception {
 
     ApiData data = new ApiData();
     SmartcarResponse res = new SmartcarResponse(data);
-    RequestBody body = new FormBody.Builder()
-      .add("action", "LOCK")
+    JsonObject json = Json.createObjectBuilder()
+      .add("action", "UNLOCK")
```

Comment:
Why is this UNLOCK now?

Reply:
Good catch! Hopefully integration tests will catch mistakes like this.